sstables/version.hh

```diff
@@ -86,6 +86,13 @@ inline seastar::sstring to_string(sstable_version_types format) {
     throw std::runtime_error("Wrong sstable format");
 }
 
+inline seastar::sstring to_string(sstable_format_types format) {
```

Comment:
how about specializing `fmt::format` for `sstable_format_types` instead? for two reasons it (apparently) will work with fmtlib. that means we can:
  - `fmt::print(ostream, "{}", format)` for printing it to a stream,
  - `logger.debug("format={}", format)` 
  - `fmt::to_string(format)` for converting it to a string

and in the first two use cases, the formatted string is not materialized, so presumably we have better performance in these use cases.

Reply:
Nice suggestion. Then the to_string(sstable_version_types) are to be converted too :) I'll try to prepare v2